src/map/entities/baseentity.cpp

```diff
@@ -153,3 +153,13 @@ uint16 CBaseEntity::GetModelId()
 {
     return look.modelid;
 }
+
+void CBaseEntity::SetLookSize(uint16 size)
+{
+    look.size = size;
+}
+
+uint16 CBaseEntity::GetLookSize()
```

Comment:
That byte in the look value on these NPCs is not really size and we should stop calling it that.